### DISCUSSION TITLE
Question: Difference between version :12-alpine and :13-alpine

### DISCUSSION THREAD

**Discussion by jokay** [prodrigestivill/docker-postgres-backup-local#101] [Q&A] [answered]
What is the difference between version `:12-alpine` and `:13-alpine`?

Should someone using PostgreSQL **12** only use the `:12-alpine` version or can `:13-alpine` be used as well?

**Comment by prodrigestivill** [comment, marked as answer]
All the images tags are based on the [official postgres docker](https://hub.docker.com/_/postgres) image tags.
The backup scripts are copied ontop of each of the avaiable tags from the official image.

So `:12-alpine` are using the pgdump from postgres version 12.x and `:13-alpine` are using the pgdump from postgres version 13.x.

**Comment by jokay** [comment]
@prodrigestivill thx for clarification :+1:
